src/h3/to_multipoly.pyx

```diff
@@ -0,0 +1,80 @@
+cimport h3lib
+from h3lib cimport H3int
+from .util cimport check_cell
+
+
+# need to define `coord2geo` here due to Cython bug:
+# https://github.com/cython/cython/issues/2745
+cdef (double, double) coord2geo(h3lib.GeoCoord c):
```

Comment:
`coord2degs`?

Reply:
I `deg` it.  :sunglasses:

Updated, but without the 's'.